File: backend/Mover.Entities/POCOEntities/Georeferencia/Ciudad.cs

namespace Mover.Entities.POCOEntities.Georeferencia
{
    public class Ciudad
    {
        public int Id { get; set; }
        public string Codigo { get; set; } = string.Empty;
        public string Nombre { get; set; } = string.Empty;
        public int DepartamentoId { get; set; }
        public Departamento? Departamento { get; set; }
    }
}

File: frontend/lib/domain/entities/georeferencia/ciudad.dart
class Ciudad {
  final int? id;
  final int? departamentoId;
  final String nombre;
  final String codigo;

  Ciudad({
    this.id,
    this.departamentoId,
    this.nombre = '',
    this.codigo = '',
  });

  factory Ciudad.fromJson(Map<String, dynamic> json) {
    return Ciudad(
        id: json['id'],
        nombre: json['nombre'],
        departamentoId: json['departamentoId'],
        codigo: json['codigo']);
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'nombre': nombre,
      'departamentoId': departamentoId,
      'codigo': codigo
    };
  }
}
